assertk/src/commonTest/kotlin/test/assertk/assertions/ListTest.kt

```diff
@@ -129,7 +131,8 @@ class ListTest {
     @Test fun containsSubList_fails_if_sublist_is_empty_and_actual_is_not_empty() {
         val emptySubList: List<String> = emptyList()
         val given: List<String> = listOf("Jason", "Jane", "Anne", "Darius", "Lee")
-        assertFails { assertThat(given).containsSubList(emptySubList) }
+        val throwable = assertFails { assertThat(given).containsSubList(emptySubList) }
+        assertIs<AssertionFailedError>(throwable)
```

Comment:
I think you can use [assertFailsWith](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/assert-fails-with.html) to keep this as one line.

I would also check `AssertionError` instead of `AssertionFailedError`, as that should remain an implementation detail imo.

Reply:
Good call; will use assertFailsWith() to check for `AssertionError` in the future - thanks!